client/src/api/documents.js

```diff
@@ -37,3 +37,21 @@ export const postDocument = (
     })
   };
 };
+
+export const putTranslation = (id, content) => {
+  const putParams = {
+    body: JSON.stringify({ content: content }),
+    method: "PUT",
+    headers: {
+      Authorization: `Bearer ${sessionStorage.getItem("token")}`,
+      "Content-Type": "application/json"
+    }
+  };
+  return fetch(`/api/translations/${id}`, putParams).then(res => {
+    if (res.status >= 200 && res.status < 300) {
```

Comment:
Hi, I would suggest that you could find out which exact HTTP status code you would want your put request to return and then check for that status code instead of checking whether it is any 2xx. For example you might want your put request to return 200 (ok) or 204 (no content)? Let me know if you'd like to discuss this :)

Reply:
Thank you Aronja, I made some changes.